eng/pipelines/coreclr/perf_slow.yml

```diff
@@ -53,7 +53,7 @@ extends:
     - stage: Build
       jobs:
       - ${{ if ne(variables['System.TeamProject'], 'public') }}:
-        - template: /eng/pipelines/runtime-perf-slow-jobs.yml@performance
```

Comment:
Is this file still needed as it seems to maybe be a copy of eng/pipelines/performance/perf-slow.yml?

Reply:
This file will be needed until we change which YAML file the pipeline definition in AzDO points at (which would be after this PR is merged).